openmrs/apps/home/extension.json

```diff
@@ -1,173 +1,189 @@
-    "url": "/implementer-interface",
-    "icon": "fa fa-pencil-square-o",
-    "order": 12,
-    "requiredPrivilege": "app:implementer-interface"
-  },
-  "atomfeedConsole": {
-    "id": "bahmni.atomfeed.console",
-    "extensionPointId": "org.bahmni.home.dashboard",
-    "type": "link",
-    "translationKey": "MODULE_LABEL_ATOMFEED_CONSOLE_KEY",
-    "url": "/atomfeed-console",
-    "icon": "fa fa-terminal",
-    "order": 13,
-    "requiredPrivilege": "app:admin"
-  },
-  "appointmentScheduling": {
-    "id": "bahmni.appointment.scheduling",
-    "extensionPointId": "org.bahmni.home.dashboard",
-    "type": "link",
-    "translationKey": "MODULE_LABEL_APPOINTMENT_SCHEDULING_KEY",
-    "url": "../appointments",
-    "icon": "fa fa-calendar",
-    "order": 14,
-    "requiredPrivilege": "app:appointments"
-  },
-  "possibleDhis2Integration": {
-    "id": "possible.dhis2Integration",
-    "extensionPointId": "org.bahmni.home.dashboard",
-    "type": "link",
-    "translationKey": "MODULE_LABEL_DHIS_2_INTEGRATION_KEY",
-    "url": "/dhis-integration/index.html",
-    "icon": "fa-book",
-    "order": 15,
-    "requiredPrivilege": "app:reports"
-  },
-  "bahmniIncidentReporting": {
-    "id": "bahmni.incidentReporting",
-    "extensionPointId": "org.bahmni.home.dashboard",
-    "type": "link",
-    "translationKey": "MODULE_LABEL_INCIDENT_REPORTING",
-    "url": "https://docs.google.com/forms/d/1k55evW5y6v8_Tg1rN8IyNXGVbv1UbtKHEoPpinjdYgk/viewform?c=0&w=1",
-    "target": "_blank",
-    "icon": "fa-flag",
-    "order": 16
-  },
-  "bahmniIssueReporting": {
-    "id": "bahmni.issueReporting",
-    "extensionPointId": "org.bahmni.home.dashboard",
-    "type": "link",
-    "translationKey": "MODULE_LABEL_REPORT_IT/EHR_ISSUE",
-    "url": "https://docs.google.com/a/possiblehealth.org/forms/d/1C-F0ZBfhEaJCJDEj_iCfw4RZwYhajJV49D0IA_tLi4w/viewform?entry.1164338174&entry.94144145&entry.598291519=@possiblehealth.org&entry.1513625367=Today&entry.1833753534",
-    "target": "_blank",
-    "icon": "fa-bug",
-    "order": 17
-  }
+	"registration": {
+		"id": "bahmni.registration",
+		"extensionPointId": "org.bahmni.home.dashboard",
+		"type": "link",
+		"translationKey": "MODULE_LABEL_REGISTRATION_KEY",
+		"url": "../registration/index.html",
+		"icon": "fa-user",
+		"order": 1,
+		"requiredPrivilege": "app:registration"
+	},
+	"clinical": {
+		"id": "bahmni.clinical",
+		"extensionPointId": "org.bahmni.home.dashboard",
+		"type": "link",
+		"translationKey": "MODULE_LABEL_CLINICAL_KEY",
+		"url": "../clinical/index.html#/default/patient/search",
+		"icon": "fa-stethoscope",
+		"order": 2,
+		"requiredPrivilege": "app:clinical"
+	},
+	"adt": {
+		"id": "bahmni.adt",
+		"extensionPointId": "org.bahmni.home.dashboard",
+		"type": "link",
+		"translationKey": "MODULE_LABEL_INPATIENT_KEY",
+		"url": "../adt/",
+		"icon": "icon-bahmni-inpatient",
+		"order": 3,
+		"requiredPrivilege": "app:adt"
+	},
 
+	"orders": {
+		"id": "bahmni.orders",
+
```

Comment:
It seems that you have deleted the contents of orders and the brackets of orders are open, so all other apps falls under the orders.
"orders" :{
},
"bahmniIpd":{},